src/app/operators/dialogs/csv/csv-config/csv-properties/csv-properties.component.ts

```diff
@@ -259,8 +248,21 @@ export class CsvPropertiesComponent implements OnInit, AfterViewInit, OnDestroy
                     this.spatialProperties.controls['yColumn'].disable();
                     this.spatialProperties.controls['wktResultType'].enable();
                     this.propertiesService.xyColumn$.next({x: this.propertiesService.xyColumn$.getValue().x});
+                    this.temporalUpdate();
                 } else {
+                    this.temporalUpdate();
+                    let x = this.spatialProperties.controls['xColumn'].value;
```

Comment:
It is not clear what is happening here, maybe you can extract a function and give it a suitable name.